phpBB/adm/style/admin.css

```diff
@@ -728,6 +728,18 @@ td {
 	border-collapse: separate;
 }
 
+.table2 {
+	display: inline-block;
```

Comment:
this leads to a weird display issue on very widescreens above 1400px where the div and table are not the same width. Try `display: table` Have not checked on windows browsers where the original issue was with firefox i believe.

Reply:
I see now what you mean, I have a Full-HD monitor.
Using Firefox / Windows here BTW. The change you proposed doesn't do the trick.

Let me see if I can fix that glitch...

Reply:
Fixed with the latest commit, here's a grab (1920 pixels)

![immagine](https://user-images.githubusercontent.com/480857/71450110-7fed8080-275b-11ea-96ba-731565935b4f.png)